test_runner/regress/test_tenant_size.py

```diff
@@ -710,3 +710,103 @@ def mask_model_inputs(x):
         return newlist
     else:
         return x
+
+
+@pytest.mark.parametrize("zero_gc", [True, False])
+def test_lsn_lease_size(neon_env_builder: NeonEnvBuilder, test_output_dir: Path, zero_gc: bool):
+    """
+    Compare a LSN lease to a read-only branch for synthetic size calculation.
+    They should have the same effect.
+    """
+
+    conf = {"pitr_interval": "0s"} if zero_gc else {"pitr_interval": "3600s"}
```

Comment:
```suggestion
    conf = {
      "pitr_interval": "0s" if zero_gc else "3600s",
      "gc_period": "0s",
    }
```

gc_period: 0s might make it so that this test doesn't become flaky, and with the current compaction work, it might also signal this test might be gc sensitive.